### ISSUE TITLE
get5 & csgo-practice-mode combo

### ISSUE THREAD

**Issue by sF0rman** [splewis/get5#553]
I'm using both plugins, practice mode for a while and added get5 yesterday. Seems to be an issue with startmoney if launching !get5 scrim while in practice mode.

### Expected behavior
Expect all practice mode settings to stop when get5 is run. Specifically reset startmoney to 800

### Actual behavior
Seems to work fine at first. warmup, ready, knife, side selection, but when game starts everyone has 16000 startmoney.
Even when I manually add rcon mp_startmoney 800 -> rcon mp_restartgame 1

### Steps to reproduce

- Plugin version: 0.7.1
- csgo-practice-mode version: 1.3.3
- Sourcemod version: 
- Steps to reproduce (please be specific):

inside server as admin do .prac -> !get5 -> start scrim from menu

I have double and triple checked configs and confirmed that they say mp_startmoney 800. Seing how both plugins are from same dev (splewis),

**Comment by sF0rman** [comment]
Double checked and made sure to disable prac mode before and during get5 scrim which worked fine, so workaround is just make sure prac mode is disabled before starting scrim. This would be a priority low fix, but nice to have if possible.

**Comment by ErcouldnT** [comment]
I wrote "sm_practicemode_can_be_started 0" in live.cfg but I can't activate prac mode after match ends. And still there is a problem like going match without turning off prac mode. @splewis

**Comment by MikkelDK** [comment]
Always double check! @ErcouldnT 

Check from this file about it's turned off: ``csgo/csgo/cfg/sourcemod/practicemode.cfg``

**Comment by ErcouldnT** [comment]
I didn't get it. I just want to turn prac mode off automatically when match starts.

// Default: "1"
sm_practicemode_can_be_started "1" in the file which you have talked about.

**Comment by MikkelDK** [comment]
Oh, the way.. I think the plugin needs to be modified. ^^

**Comment by richanynguon** [comment]
For anyone who is currently having this same issue my work around is:

To adjust these lines or create:
in `csgo/cfg/sourcemod/practicemode.cfg` 
```
sm_practicemode_autostart "0" 
```

in `csgo/cfg/get5/knife.cfg` and `csgo/cfg/get5/live.cfg`
```
sm_practicemode_can_be_started "0" 
```

in `csgo/cfg/prac.cfg` (would have to create)
```
sm_practicemode_can_be_started "1" 
```

in `csgo/cfg/server.cfg` 
```
rcon_password "yourverysecurepassword"
```

The catch to this solution is that:
1.  Every server instance requires a get5 or prac selection
2. Creating a successful get5 instance has a pre-requirement is to turn off prac mode 
3. To turn prac mode back on after get5, would require to use rcon in console: `rcon_password yourverysecurepassword; rcon exec prac;`

**Comment by PhlexPlexico** [comment]
I think another fairly easy fix instead of using rcon would be to place `sm_practicemode_can_be_started "1" ` inside another CFG file of your choosing, and using [get5_autoload_config](https://splewis.github.io/get5/configuration/#get5_autoload_config) to load that file, which loads whenever there is no match (which happens after the match delay finishes). Even if practice mode isn't in there, it would just complain that it's not a command either.
